source/dem/force_chains_visualization.cc

```diff
@@ -79,9 +79,8 @@ ParticlesForceChains<dim, contact_model, rolling_friction_model>::
 {
   auto &local_adjacent_particles = container_manager.local_adjacent_particles;
   // Lines 89 to 101 kept for future ghost particles implementation.
-  // auto &ghost_adjacent_particles =
-  // container_manager.ghost_adjacent_particles; auto
-  // &local_periodic_adjacent_particles =
+  auto &ghost_adjacent_particles = container_manager.ghost_adjacent_particles;
+  // auto &local_periodic_adjacent_particles =
```

Comment:
I think we can delete those commented line since we don't know when it is gonna be implemented. 

In the last PR, I though those commented lines were fine since the local_ghost PR we going to be implemented quickly, but for the periodic we don't have a timeline yet.